src/core/workload/kyuubi.py

```diff
@@ -17,16 +19,26 @@
     KYUUBI_SERVICE_NAME,
 )
 from core.domain import User
-from core.workload import KyuubiWorkloadBase
+from core.workload import KyuubiPaths, KyuubiWorkloadBase
 from utils.logging import WithLogging
 
+KYUUBI_CONF_PATH = "/opt/kyuubi/conf"
+SPARK_CONF_PATH = "/etc/spark8t/conf"
+KYUUBI_ROOT = "/opt/kyuubi"
+
 
 class KyuubiWorkload(KyuubiWorkloadBase, K8sWorkload, WithLogging):
     """Class representing workload implementation for Kyuubi on K8s."""
 
     def __init__(self, container: Container, user: User = User()):
```

Comment:
**out-of-scope** not for this PR, but I found it kinda weird that we have something that is substrate specific (the container) in the `core` module, which should be more general. 

Probably in Kyuubi does not matter too much since we only support K8s, so it is not too critical, but coming here from other projects, it got me a bit confused at first. Having a more common pattern would help people find the things straight away, even if they don't know Kyuubi. Anyhow, non-blocking and up to you to decide whether it would be worth it to follow-up (evaluating tradeoff between overenginnering and a strucutre that is more compliant and similar to other charms)